examples/specs/window_residual_graph.vl.json

```diff
@@ -8,20 +8,20 @@
     }
   },
   "transform": [
-      {"filter": "datum.IMDB_Rating != null"},
-      {"filter": {"timeUnit": "year", "field": "Release_Date", "range": [null, 2019]}},
-      {
-        "window": [{
-            "op": "mean",
-            "field": "IMDB_Rating",
-            "as": "AverageRating"
-        }],
-        "frame": [null, null]
-      },
-      {
-        "calculate": "datum.IMDB_Rating - datum.AverageRating",
-        "as": "RatingDelta"
-      }
+    {"filter": "datum.IMDB_Rating != null"},
```

Comment:
Why are these tabbings related to build changes?

Reply:
They are not. I got an error in the file and saw it.